src/test/lib/logical_query_plan/lqp_translator_test.cpp

```diff
@@ -609,7 +609,7 @@ TEST_F(LQPTranslatorTest, ProjectionNode) {
   EXPECT_EQ(*projection_op->expressions[0], *PQPColumnExpression::from_table(*table_int_float, "a"));
 }
 
-TEST_F(LQPTranslatorTest, JoinNodeToJoinHash) {
+TEST_F(LQPTranslatorTest, DISABLED_JoinNodeToJoinHash /* TMP: #2321 */) {
```

Comment:
?

Reply:
Just to trigger everything in the CI. Does not pass as we need to have SMJ for the bug. This test checks for the hash join. Will be reverted.